src/smexperiments/tracker.py

```diff
@@ -539,10 +532,7 @@ def log_roc_curve(
         output_artifact=True,
     ):
         """Log a receiver operating characteristic (ROC curve) artifact which will be displayed in
-        studio.  Requires sklearn.
-
-        Note that this method must be run from a SageMaker context such as studio or training job
-        due to restrictions on the CreateArtifact API.
+        Studio's Pipeline UI (not the Trial Component UI).  Requires sklearn.
```

Comment:
Im a bit confused here by:

1) what if I don't use these functions for pipeline execution, say, just for a single training job.
2) users are using experiment SDK, mentioning pipeline without more context is a bit confusing tho.

Reply:
1. the association and artifact will be created but there will be no way to visualize the graph from within Studio (for the time being)
2. ya, user may not know about pipelines or how they are rendered in Studio. maybe a link to a pipeline's blog post would provide context.

Reply:
yea true ... probably better to give more details here, e.g "if ur TC is created by/within a pipeline execution context, you can view the table from the step (who generates the TC) view." .... and a link.